src/dbos-workflow.ts

```diff
@@ -56,19 +56,19 @@ export interface OperonConfig {
 }
 
 interface WorkflowInfo<T extends any[], R> {
-  workflow: OperonWorkflow<T, R>;
+  workflow: DBOSWorkflow<T, R>;
   config: WorkflowConfig;
 }
 
-export class Operon {
+export class DBOSWFE {
```

Comment:
WFE is a bit abstract and the original Operon does more than workflows.

```suggestion
export class DBOSExecutor {
```